packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -210,6 +211,17 @@ abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   /// If null then [kMaxFlingVelocity] is used.
   double? maxFlingVelocity;
 
+  /// Whether the drag threshold should be met before dispatching any drag callbacks.
```

Comment:
Nit: Should you explain what "meeting the drag threshold" means?  Or maybe link to `touchSlop`?

Reply:
Done. I linked `computeHitSlop` and `computePanSlop` as they determine the threshold.